packages/js-example-app/src/index.ejs

```diff
@@ -133,6 +133,19 @@
             <p label="Tab 11" icon="blr360">Tab 11</p>
           </blr-tab-bar>
         </div>
+
+        <div class="component">
+          <p>Toggle Switch</p>
+          <blr-label-toggleswitch
+            theme="Light"
+            size="md"
+            showstatelabel="true"
```

Comment:
should be kebab-case.

Reply:
Ok, I think that the kebab-case changes that you made a few weeks ago didn't take effect on the code snippet part of storybook. That's where I lifted this code from. We might need to fix that at a later time